paddle/pten/api/lib/tensor.cc

```diff
@@ -214,17 +214,22 @@ const T *Tensor::data() const {
 template PD_DLL_DECL const float *Tensor::data<float>() const;
 template PD_DLL_DECL const double *Tensor::data<double>() const;
 template PD_DLL_DECL const int64_t *Tensor::data<int64_t>() const;
+template PD_DLL_DECL const uint64_t *Tensor::data<uint64_t>() const;
```

Comment:
why need uint64 and uint32? If there is no current demand, over-design is not recommended